frontend/app/routes/$lang+/_protected+/letters+/index.tsx

```diff
@@ -106,7 +106,7 @@ export default function LettersIndex() {
 
           return (
             <li key={letter.id} className="py-4 sm:py-6">
-              <InlineLink reloadDocument to={`/letters/${letter.referenceId}/download`} className="font-lato font-semibold">
+              <InlineLink reloadDocument to={`/letters/${letter.referenceId}/download`} className="font-lato font-semibold" target="_blank" rel="norefferer">
```

Comment:
I feel there's no need for the `rel="norefferer"` when linking to another page within our application.

> The noreferrer keyword for the `rel` attribute instructs the browser, when navigating to the target resource, to omit the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) header and otherwise leak no referrer information

See https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer

```suggestion
              <InlineLink reloadDocument to={`/letters/${letter.referenceId}/download`} className="font-lato font-semibold" target="_blank">
```